src/azure-cli/azure/cli/command_modules/appconfig/_kv_import_helpers.py

```diff
@@ -0,0 +1,958 @@
+            for flag in ms_feature_flags:
+                index_of_existing_flag = next((i for i, existing_flag in enumerate(feature_flags) if existing_flag.id == flag.id), -1)
+
+                if index_of_existing_flag != -1:
+                    feature_flags[index_of_existing_flag] = flag
+                else:
+                    feature_flags.append(flag)
+
+        for feature in feature_flags:
+            key = FeatureFlagConstants.FEATURE_FLAG_PREFIX + str(feature.id)
+            set_kv = KeyValue(
+                key=key,
+                value=json.dumps(
+                    feature,
+                    default=lambda o: {k: v for k, v in o.__dict__.items() if v is not None},
+                    ensure_ascii=False,
+                ),
+                content_type=FeatureFlagConstants.FEATURE_FLAG_CONTENT_TYPE,
+            )
+            key_values.append(set_kv)
+
+    except Exception as exception:
+        raise CLIError(
+            "File contains feature flags in invalid format. " + str(exception)
+        )
+    return key_values
+
+
+def __get_ms_feature_management_schema(features_dict):
+    ms_fm_keyword = FeatureManagementReservedKeywords.UNDERSCORE.feature_management
+    feature_management_section = features_dict.get(ms_fm_keyword, None)
+    if feature_management_section and not isinstance(feature_management_section, dict):
+        raise ValidationError("Feature management section must be a dictionary.")
+
+    if not isinstance(feature_management_section[FeatureFlagConstants.FEATURE_FLAGS_KEY], list):
+        raise ValidationError("Feature flags must be a list.")
+
+    return feature_management_section
+
+
+def __get_dotnet_feature_management_schema(
+    dotnetSchemaFeatureManagementKeyWord, features_dict
+):
+    ms_fm_keyword = FeatureManagementReservedKeywords.UNDERSCORE.feature_management
+    feature_management_section = features_dict.get(
+        dotnetSchemaFeatureManagementKeyWord, None
+    )
+
+    if feature_management_section and not isinstance(feature_management_section, dict):
+        raise ValidationError("Feature management section must be a dictionary.")
+
+    if dotnetSchemaFeatureManagementKeyWord == ms_fm_keyword:
+        return dict(
+            (key, feature_management_section[key])
+            for key in feature_management_section.keys()
+            if key != FeatureFlagConstants.FEATURE_FLAGS_KEY
+        )
+    return feature_management_section
+
+
+def __read_features_from_msfm_schema(feature_flags_list):
+    feature_flags = []
+    for feature in feature_flags_list:
+        if validate_import_feature(feature[FeatureFlagConstants.ID]):
+            feature_id = feature[FeatureFlagConstants.ID]
+
+            new_feature = FeatureFlagValue(
+                id_=str(feature.get(FeatureFlagConstants.ID)),
+                enabled=_convert_boolean_value(
+                    feature.get(FeatureFlagConstants.ENABLED, None), feature_id
```

Comment:
if "enabled" is not present, default to false

Reply:
That's true for all boolean properties in FM schema

Reply:
Set default to False